forge-gui/res/cardsfolder/upcoming/kagha_shadow_archdruid.txt

```diff
@@ -0,0 +1,11 @@
+Name:Kagha, Shadow Archdruid
+ManaCost:2 B G
+Types:Legendary Creature Elf Druid
+PT:1/4
+T:Mode$ Attacks | ValidCard$ Card.Self | Execute$ TripPump | Subability$ TrigMill TriggerDescription$ When CARDNAME attacks, it gains deathtouch until end of turn. Mill two cards. (Put the top two cards of your library into your graveyard.)
```

Comment:
TripPump = TrigPump
Subability$ TrigMill TriggerDescription$ = Subability$ TrigMill | TriggerDescription$

Reply:
Test it and see? The trigger shouldn't have worked at all before those fixes.